guides/micronaut-jaxrs-jdbc/micronaut-jaxrs-jdbc.adoc

```diff
@@ -0,0 +1,94 @@
+common:header.adoc[]
+
+The application exposes some REST endpoints using https://projects.eclipse.org/projects/ee4j.jaxrs[JAX-RS annotations] and stores data in a MySQL database using Micronaut Data JDBC.
```

Comment:
need to explain what JAX-RS is, why they'd use it, why they'd use it instead of what's available in Micronaut, point out the micronaut-jaxrs docs, etc.